server.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express')
+const mongoose = require('mongoose')
+const app = express()
+const ShortUrl = require('./models/shortUrl')

```

Comment:
There is no `models` directory in this pull request. Did you forgot to create it and move `shortUrl.js` into it or did you mean the following require here:

```suggestion
const ShortUrl = require('./shortUrl')
```